MapboxCoreNavigation/RouteController.swift

```diff
@@ -230,7 +230,13 @@ open class RouteController: NSObject, Router {
      - important: If the rawLocation is outside of the route snapping tolerances, this value is nil.
      */
     var snappedLocation: CLLocation? {
-        self.rawLocation?.snapped(to: self.routeProgress.currentLegProgress)
+        let snappedDefault = self.rawLocation?.snapped(to: self.routeProgress.currentLegProgress)
```

Comment:
Aren't you doing duplicated work here? If there is a delegate, the delegate should do the snapping, otherwise the default snapping happens.